src/main/java/de/caritas/cob/agencyservice/api/service/AgencyService.java

```diff
@@ -185,10 +199,32 @@ private List<Agency> findAgencies(AgencySearch agencySearch) {
               TenantContext.getCurrentTenant());
     } catch (DataAccessException ex) {
       throw new InternalServerErrorException(LogService::logDatabaseError,
-          "Database error while getting postcodes");
+          DB_POSTCODES_ERROR);
     }
   }
 
+  private List<Agency> findAgencies(String postCode, Integer topicId) {
```

Comment:
this is not saying that it's for the current tenant, probably worth reflecting it in the name